ssc/Pos/Arbitrary/Ssc.hs

```diff
@@ -38,9 +38,11 @@ import           Pos.Util.QuickCheck.Arbitrary (Nonrepeating (..), sublistN)
 -- Types
 ----------------------------------------------------------------------------
 
+{-
 newtype SscPayloadDependsOnSlot = SscPayloadDependsOnSlot
     { genPayloadDependsOnSlot :: SlotId -> Gen SscPayload
     } deriving Generic
+-}
```

Comment:
Like the export above. Should either delete or uncomment it and actually export it.